src/pages/slide.astro

```diff
@@ -0,0 +1,376 @@
+---
+import DocumentationLayout from '../layouts/documentation.astro'
+import { Markdown } from 'astro/components'
+import Button from '../components/button.astro'
+import ArrowIcon from '../components/icons/arrow.astro'
+import CodeDemo from '../components/code-demo.astro'
+
+const accordionItems = [
+  {
+    heading: 'this is a heading',
+    text: 'this is the description that you had to expand!'
+  },
+  {
+    heading: 'this is another heading',
+    text: 'asdf asdfasdf this is the description that you had to expand!'
+  },
+  {
+    heading: 'asdfasdfasdfasdfa this is another heading',
+    text: 'qwerqwe this is the description that you had to expand!'
+  },
+]
+---
+<DocumentationLayout title="Slide">
+  <section slot="api">
+    <Markdown>
+      ## Install
+
+      ```js
+      // index.js
+      import SlideController from '@viget/stimulus-controllers/slide_controller'
+      Stimulus.register('slide', SlideController)
+      ```
+
+      ## Values 
+
+      | Value | Full Attribute | Type | Default | Description |
+      |:---|:---|:---|:---|:---|
+      | `isOpen` | `data-slide-is-open-value` | Boolean | `false` |  Controls whether the `receiver` is expanded or collapsed |
+
+      ## Targets
+
+      | Target | Full Attribute | Description | Required |
+      |:---|:---|:---|:---|
+      | `toggler` | `data-slide-target="toggler"` | Add to the elements using `toggle`, `open`, or `close`. This should only be used if there are multiple elements with `aria-expanded` that need to be synced; otherwise the element is grabbed from the event. | No |
+      | `receiver` | `data-slide-target="receiver"` | Add to the element that is being toggled/opened/closed. This is used in almost all patterns, but if not set it will default to the toggler element. | No |
+
+      ## Classes
+
+      | Class | Full Attribute | Description |
+      |:---|:---|:---|
+      | `collapsed` | `data-slide-collapsed-class` | This defaults to `collapsed`. Use cases for overriding might be a responsive version (e.g. `sm:collapsed`), a different animation, or a different `min-height`. See [CSS](#css) for more information. |
+
+      ## Actions
+
+      ### `toggle()`
+      _Event: `slide:toggle`_
+
+      Slides `receiver` target open or closed by toggling `isOpen` value
+
+      ---
+
+      ### `open()`
+      _Event: `slide:open`_
+
+      Opens `receiver` target by setting `isOpen` value to `true`
+
+      ---
+
+      ### `close()`
+      _Event: `slide:close`_
+
+      Closes `receiver` target by setting `isOpen` value to `false`
+
+      ## Dependencies
+
+      ### CSS
```

Comment:
Call me crazy... but what if skipped any depenencies and managed the styling / transitions within the controller and/or `Slide`.

Transition durations & timing functions could be optional Stimulus values.

Reply:
interestingggg. one thing i like about providing the class is you can easily do other custom things with the CSS -- e.g. use a diff collapsed `max-height` & a pseudo element for more of a collapsed-but-here's-a-peek thing. i think more important than that would be media query support, but could do something like i set up on `tabs` for passing that as a value. 

i do appreciate the idea of eliminating deps, tho

Reply:
Such a tricky balance. It would be so rad to just drop in this controller and "bam" you've got a pretty cool transition that works for 80% of your use cases.

Downside to having the default transition be fully in the domain of the controller is that changing the default for every instance would take some sort of global config or something. 

Your solution of having a default class of`.collapsed` seems like it might be the best of both world. Giving end users a way to modify things globally just in their CSS or Tailwind plugin.